include/ola/acn/ACNFlags.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * ACNFlags.h
+ * Flags used in ACN PDUs
+ * Copyright (C) 2020 Peter Newman
```

Comment:
Believe it or not, I've been working on bits of it (like this bit) since then!:
https://github.com/OpenLightingProject/ola/commit/e9dd3742f9310c0a72bccacfc45cfb9bacf115aa

Given we normally just keep the original copyright date I think I'll leave it as is.

Reply:
I thought right from the start that it was the date you started with it :) Sure, if the habit is to keep them, I don't have an issue with that :+1:

Reply:
Well we certainly don't normally touch them when we change the file. I figure this isn't very different.